procrustes/test/test_psdp.py

```diff
@@ -43,21 +46,22 @@ def test_psdp_diagonal():
     a = np.diag([1, 2, 3, 4])
     b = np.eye(4)
     res = psdp_woodgate(a=a, b=b)
-    s = res["s"]
-    error = res["error"]
+    s, error = res["s"], res["error"]
     actual_result = np.diag([0.99999, 0.5, 0.33333, 0.25])
     assert_almost_equal(s, actual_result, decimal=5)
     assert_almost_equal(error, 0.0, decimal=3)
+    res = psdp_peng(a=a, b=b)
+    s, error = res["s"], res["error"]
+    assert_almost_equal(s, actual_result, decimal=5)
+    assert_almost_equal(error, 0.0, decimal=3)
 
 
 def test_psdp_generic_square():
     r"""Test PSDP with 2 generic square matrices."""
-    # The example given here is from the original paper.
     a = np.array([[1, 6, 0], [4, 3, 0], [0, 0, -0.5]])
     b = np.array([[1, 0, 0], [0, -2, 3], [0, 2, 4]])
     res = psdp_woodgate(a=a, b=b)
-    s = res["s"]
-    error = res["error"]
+    s, error = res["s"], res["error"]
     actual_result = np.array(
         [
             [0.22351489, -0.11059539, 0.24342428],
```

Comment:
The same as above.

Reply:
fixed in [8c7015b](https://github.com/theochem/procrustes/pull/187/commits/8c7015b5413c07718ecf7cd817f58568144fa3ae)